docs/TEMPLATES.md

```diff
@@ -463,9 +463,7 @@ Markdown formatted texts are converted to HTML with one of these libraries:
   * Kramdown - `Tilt::KramdownTemplate`
   * Pandoc - `Tilt::PandocTemplate`
   * Maruku - `Tilt::MarukuTemplate`
-
-Tilt will use fallback mode (as documented in the README) for determining which
```

Comment:
There doesn't seem to be any other mention of `fallback` anywhere else in the codebase.